atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DbAtlasDbFactory.java

```diff
@@ -38,8 +38,10 @@ public String getType() {
         return TYPE;
     }
 
+    // TODO(gmaretic): async initialization not implemented/propagated
     @Override
-    public KeyValueService createRawKeyValueService(KeyValueServiceConfig config, Optional<LeaderConfig> leaderConfig) {
+    public KeyValueService createRawKeyValueService(KeyValueServiceConfig config, Optional<LeaderConfig> leaderConfig,
+            boolean initializeAsync) {
```

Comment:
Do we have to pass `initializeAsync` here?

Reply:
Same as above.